addons/google_calendar/models/google_calendar.py

```diff
@@ -834,20 +811,20 @@ def update_events(self, cr, uid, lastSync=False, context=None):
                         if event.GG.status:
                             parent_event = {}
                             if not event_to_synchronize[base_event][0][1].OE.event_id:
-                                main_ev = att_obj.search_read(cr, uid, [('google_internal_event_id', '=', event.GG.event['id'].rsplit('_', 1)[0])], fields=['event_id'], context=context_novirtual)
-                                event_to_synchronize[base_event][0][1].OE.event_id = main_ev[0].get('event_id')[0]
+                                main_ev = CalendarAttendee.with_context(context_novirtual).search([('google_internal_event_id', '=', event.GG.event['id'].rsplit('_', 1)[0])], limit=1)
+                                event_to_synchronize[base_event][0][1].OE.event_id = main_ev.event_id.id
 
                             parent_event['id'] = "%s-%s" % (event_to_synchronize[base_event][0][1].OE.event_id, new_google_event_id)
-                            res = self.update_from_google(cr, uid, parent_event, event.GG.event, "copy", context)
+                            res = recs.update_from_google(parent_event, event.GG.event, "copy")
                         else:
                             parent_oe_id = event_to_synchronize[base_event][0][1].OE.event_id
                             if parent_oe_id:
-                                calendar_event.unlink(cr, uid, "%s-%s" % (parent_oe_id, new_google_event_id), can_be_deleted=True, context=context)
+                                CalendarEvent.browse("%s-%s" % (parent_oe_id, new_google_event_id)).unlink(can_be_deleted=True)
```

Comment:
I suppose unlink should be called with "with_context(curr_attendee=event.OE.attendee_id)"